invenio_accounts_rest/ext.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""REST API for invenio-accounts."""
+
+from __future__ import absolute_import, print_function
+
+from flask_babelex import gettext as _
+
+from .views import blueprint
+
+
+class InvenioAccountsREST(object):
+    """Invenio-Accounts-REST extension."""
+
+    def __init__(self, app=None):
+        """Extension initialization."""
+        # TODO: This is an example of translation string with comment. Please
```

Comment:
You will need to run a ``git grep TODO`` on the repo and fix the issues. Also, all the template stuff can all be removed.